src/platform/backends/lxd/lxd_virtual_machine.cpp

```diff
@@ -487,7 +487,16 @@ void mp::LXDVirtualMachine::add_network_interface(int index, const mp::NetworkIn
 
     QJsonObject patch_json{{"devices", QJsonObject{{net_name, net_config}}}};
 
-    lxd_request(manager, "PATCH", url(), patch_json);
```

Comment:
To go with my `try`/`catch` idea, I think letting the exception through here like before would be more desirable unless I'm missing something.

Reply:
Definitely. Reverted.